cwms-data-api/src/main/java/cwms/cda/data/dao/project/ProjectDao.java

```diff
@@ -383,6 +450,12 @@ public static BigInteger toBigInteger(int revokeTimeout) {
     }
 
 
+    /**
+     * Retrieves the locations associated with a project.
+     *
+     * @param office The office ID associated with the project.
+     * @return A list of Location objects representing the project's locations.
+     */
```

Comment:
down below on: 418/491 builder.withTimeZoneName(ZoneId.of(timeZoneName)); Not all Oracle time zones map to Java time zones. @adamkorynta and I added in extra time zone coverage via Zone's map API in one of our APIs (nucleus?). That same business should occur with time zones in CDA.

Reply:
It's in OracleTypeMap::toZoneId:

https://bitbucket.hecdev.net/projects/CWMS/repos/hec-cwms-data-access/browse/cwms-db-dao/src/main/java/usace/cwms/db/dao/util/OracleTypeMap.java#397

Reply:
done

Reply:
done